ArduPlane/takeoff.cpp

```diff
@@ -155,11 +155,12 @@ void Plane::takeoff_calc_roll(void)
 void Plane::takeoff_calc_pitch(void)
 {
     if (auto_state.highest_airspeed < g.takeoff_rotate_speed) {
-        // we have not reached rotate speed, use a target pitch of 5
-        // degrees. This should be enough to get the tail off the
-        // ground, while making it unlikely that overshoot in the
-        // pitch controller will cause a prop strike
-        nav_pitch_cd = 500;
+        // we have not reached rotate speed, use the specified
+        // takeoff target pitch angle and stop the integrator from winding up
+        nav_pitch_cd = int32_t(100.0f * mode_takeoff.ground_pitch);
+        
+        // Set the pitch integrator to give a specified percentage trim.
```

Comment:
i think this should be a trim override, not an I set

Reply:
the integrator stuff was removed

Reply:
@magicrub

Reply:
I'm listening...